examples/ex9p.cpp

```diff
@@ -551,6 +666,45 @@ FE_Evolution::FE_Evolution(ParBilinearForm &_M, ParBilinearForm &_K,
    M_solver.SetPrintLevel(0);
 }
 
+// Implementation of class FE_Evolution
+FE_Evolution::FE_Evolution(ParBilinearForm &_M, ParBilinearForm &_K,
+                           const Vector &_b, const AIR_parameters &_AIR)
```

Comment:
This also has a lot of code duplication with the other constructor. See my above comment about using a flag rather than two constructors. This would greatly reduce the code duplication.